MIGRATING_FROM_V1.md

```diff
@@ -0,0 +1,262 @@
+# Migration guide from Trampoline V1 script
+
+This guide is solely for migrating from Trampoline V1 script. We
+recommend that you read README.md before diving into this guide.
+
+In this guide, we'll use
+[getting-started-python](https://github.com/GoogleCloudPlatform/getting-started-python)
+as an example repository.
+
+## The first step - local run
+
+The first step of the migration is to run the tests locally with
+Trampoline V2 script. Copy trampoline_v2.sh into your repository. A
+natural choice is to place the script alongside with the
+`trampoline.sh`.
+
+Here is an exmaple with our example repo. Make sure it has an
+executable bit.
+
+```sh
+$ git clone git@github.com:GoogleCloudPlatform/getting-started-python.git
+$ cd getting-started-python
+$ cp ../docker-ci-helper/trampoline_v2.sh .kokoro
+$ ls -al .kokoro/trampoline_v2.sh
```

Comment:
suggestion: curl from GitHub (+chmod) rather than needing to clone the repo? Maybe in the future we tag releases and can curl for the tagged version?

Reply:
Sure, done